test/commands/add/activity_spec.rb

```diff
@@ -51,5 +51,301 @@
+          let(:content) do
+            <<-FILE
+### Activities:
+- 2019-01-01: Went to _Paris_
+- 2018-01-01: Went to _Berlin_
+- 2017-01-01: Relocated to _Paris_
+
+### Notes:
+
+### Friends:
+
+### Locations:
+FILE
+          end
+
+          it  "prints 'Default location from [ADDED ACTIVITY DATE] to " \
+              "[NEXT DIFFERENT DEFAULT LOCATION ACIVITY DATE] set to [LOCATION]'" do
+            output = 'Default location from 2009-01-01 to 2018-01-01 set to: "Paris"'
+            assert_default_location_output(output)
+          end
+        end
+      end
+
+      describe "when preceding default location is the same" do
+        let(:preceding_activity) { "1999-01-01: Went to _Paris_" }
+
+        describe "when following default location is the same" do
+          let(:following_activity) { "2019-01-01: Relocated to _Paris_" }
+
+          it  "prints 'Default location from " \
+              "[PRECEDING ACTIVITY DATE] to present already set to [LOCATION]'" do
+            output = 'Default location from 1999-01-01 to present already set to: "Paris"'
+            assert_default_location_output(output)
+          end
+        end
+
+        describe "when following default location is different" do
+          let(:following_activity) { "2019-01-01: Relocated to _Berlin_" }
+
+          it  "prints 'Default location from " \
+              "[PRECEDING ACTIVITY DATE] to " \
+              "[FOLLOWING ACTIVITY DATE] set to [LOCATION]'" do
+            output = 'Default location from 1999-01-01 to 2019-01-01 already set to: "Paris"'
+            assert_default_location_output(output)
+          end
+        end
+      end
+
+      describe "when preceding default location is different" do
+        let(:preceding_activity) { "1999-01-01: Went to _Berlin_" }
+
+        describe "when following default location is the same" do
+          let(:following_activity) { "2019-01-01: Relocated to _Paris_" }
+
+          it "prints 'Default location from [ADDED ACTIVITY DATE] to present set to [LOCATION]'" do
+            output = 'Default location from 2009-01-01 to present set to: "Paris"'
+            assert_default_location_output(output)
+          end
+        end
+
+        describe "when following default location is different" do
+          let(:following_activity) { "2019-01-01: Relocated to _Berlin_" }
+
+          it  "prints 'Default location from " \
+              "[ADDED ACTIVITY DATE] to " \
+              "[FOLLOWING ACTIVITY DATE] set to [LOCATION]'" do
+            output = 'Default location from 2009-01-01 to 2019-01-01 set to: "Paris"'
+            assert_default_location_output(output)
+          end
+        end
+      end
+
+      describe "when activities are out of order" do
+        let(:content) do
+          <<-FILE
+### Activities:
+- 2018-01-01: Went to _Berlin_
+- 2019-01-01: Went to _Paris_
+
+
+### Notes:
+
+### Friends:
+
+### Locations:
+FILE
+        end
+
+        it  "prints 'Default location from " \
```

Comment:
Would you mind changing this to something more specific to the behavior we're trying to test? It should sort of "finish the sentence" started by the `describe` description. I'm thinking something along the lines of:

```
describe "when activities are out of order" do
  ...

  it "uses the sorted order for determining output" do
    ...
  end
end
```

since the actual format of what it prints isn't what this test is getting at.

Does that make sense?